draco.lua

```diff
@@ -0,0 +1,107 @@
+project "draco"
+
+dofile(_BUILD_DIR .. "/static_library.lua")
+
+configuration { "*" }
+
+uuid "0A414681-67D1-46F8-9405-94E446B301D3"
```

Comment:
Was this a unique uuid or was this copy pasted from a different lua file?

Reply:
I generated it